src/FormatNumber/Locales.elm

```diff
@@ -1,13 +1,12 @@
-module FormatNumber.Locales exposing (..)
+module FormatNumber.Locales exposing (Locale, frenchLocale, spanishLocale, usLocale)
 
-{-|
-
-These locales and its names are based on this
+{-| These locales and its names are based on this
 [International Language Environments
 Guide](https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html)
 
 @docs Locale
 
+
```

Comment:
Any special reason for this extra line here?

Reply:
No, it’s probably just that I used a newer version of elm-format.